conans/client/printer.py

```diff
@@ -26,9 +27,17 @@ def __init__(self, out):
     def print_inspect(self, inspect):
         for k, v in inspect.items():
             if isinstance(v, dict):
-                self._out.writeln("%s" % k)
+                self._out.writeln("%s:" % k)
                 for sk, sv in sorted(v.items()):
                     self._out.writeln("    %s: %s" % (sk, str(sv)))
+            elif k == "default_options" and v is not None:
```

Comment:
This could be simplified, if it was checked before the ``isinstance(v, dict)`` and transformed directly to a dict, then let the same as options apply to it.

Reply:
simplified, please review again.